Makefile

```diff
@@ -56,7 +56,10 @@ install-tools:
 .PHONY: install-tools
 
 lint:
-	$(ROOT_DIR)/scripts/lint.sh
+	$(ROOT_DIR)/scripts/ci/errcheck.sh
```

Comment:
may be we can use dependency targets?
`lint: ci-errcheck ci-gofmt ci-govet ci-lint`

Reply:
done, thx